llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -7056,14 +7056,11 @@ bool BoUpSLP::areAllUsersVectorized(
 
 static std::pair<InstructionCost, InstructionCost>
 getVectorCallCosts(CallInst *CI, FixedVectorType *VecTy,
-                   TargetTransformInfo *TTI, TargetLibraryInfo *TLI) {
+                   TargetTransformInfo *TTI, TargetLibraryInfo *TLI,
+                   ArrayRef<Type *> VecTys) {
```

Comment:
(pedantic) Should this be called ArgTys since we're potentially passing scalar ops as well now?